configs/forseti_conf.yaml.in

```diff
@@ -125,6 +125,8 @@ scanner:
           enabled: true
         - name: cloudsql_acl
           enabled: true
+        - name: fw_rule
```

Comment:
nit: I like to see ```firewall``` written out here for clarity, as fw is not a standard abbreviation, and to disambiguate from forwarding rule which conceivably can also be 'fw'.